src/platform/bitboxbase/leds.h

```diff
@@ -12,6 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#ifndef __BITBOXBASE_LEDS_H
+#define __BITBOXBASE_LEDS_H
+
+#include <stdbool.h>
 typedef enum { SMALL_LED, BIG_LED } led_t;
 
+typedef enum { LED_WHITE, LED_RED, LED_GREEN, LED_BLUE, LED_NONE } led_color_t;
+
 void leds_init(void);
+
+void turn_small_led(int led, bool level);
```

Comment:
`bool level` sounds a bit awkard to me. I would change the variable name to `bool enabled`.

Reply:
yeah that's a leftover. Removed.